WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -97,54 +100,61 @@ class NoticePresenter {
     // MARK: - Events
 
     private func listenToKeyboardEvents() {
-        NotificationCenter.default.addObserver(forName: UIResponder.keyboardWillShowNotification, object: nil, queue: nil) { [weak self] (notification) in
-            guard let self = self,
-                let userInfo = notification.userInfo,
-                let keyboardFrameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue,
-                let durationValue = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? NSNumber else {
-                    return
-            }
-
-            self.currentKeyboardPresentation = .present(height: keyboardFrameValue.cgRectValue.size.height)
-
-            guard let currentContainer = self.currentNoticePresentation?.containerView else {
-                return
-            }
+        NotificationCenter.default
+            .publisher(for: UIResponder.keyboardWillShowNotification)
+            .sink { [weak self] (notification) in
+                guard let self = self,
+                    let userInfo = notification.userInfo,
+                    let keyboardFrameValue = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue,
+                    let durationValue = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? NSNumber else {
+                        return
+                }
 
-            UIView.animate(withDuration: durationValue.doubleValue, animations: {
-                currentContainer.bottomConstraint?.constant = self.onScreenNoticeContainerBottomConstraintConstant
-                self.view.layoutIfNeeded()
-            })
-        }
-        NotificationCenter.default.addObserver(forName: UIResponder.keyboardWillHideNotification, object: nil, queue: nil) { [weak self] (notification) in
-            self?.currentKeyboardPresentation = .notPresent
+                self.currentKeyboardPresentation = .present(height: keyboardFrameValue.cgRectValue.size.height)
 
-            guard let self = self,
-                let currentContainer = self.currentNoticePresentation?.containerView,
-                let userInfo = notification.userInfo,
-                let durationValue = userInfo[UIResponder.keyboardAnimationDurationUserInfoKey] as? NSNumber else {
+                guard let currentContainer = self.currentNoticePresentation?.containerView else {
                     return
+                }
+
+                UIView.animate(withDuration: durationValue.doubleValue, animations: {
+                    currentContainer.bottomConstraint?.constant = self.onScreenNoticeContainerBottomConstraintConstant
+                    self.view.layoutIfNeeded()
+                })
             }
+            .store(in: &notificationObservers)
+        NotificationCenter.default
```

Comment:
What do you think of adding an empty line here to visually separate the two `NotificationCenter` operations?

```suggestion
            .store(in: &notificationObservers)

        NotificationCenter.default
```